src/hexo/themes/documentation/layout/partials/contributors.hbs

```diff
@@ -0,0 +1,25 @@
+<main class="container">
+    <article class="content" aria-labelledby="page-heading">
+        <nav class="container breadcrumb" aria-label="breadcrumbs">
+            <ul>
+                <li><a href="/">Home</a></li>
+                <li><a href="/">About</a></li>
```

Comment:
We should have an about page in case JS is not working so people can navigate to somewhere and still access the different sections. Similar to what we did with https://sonarwhal.com/docs/